x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/user/TransportDeleteUserActionTests.java

```diff
@@ -269,4 +230,14 @@ public void onFailure(Exception e) {
         assertThat(throwableRef.get(), sameInstance(e));
         verify(usersStore, times(1)).deleteUser(eq(request), anyActionListener());
     }
+
+    private String randomInternalUsername() {
```

Comment:
This gets pretty repetitive but it feels like too specific a thing to include in ESTestCase/create a util for

Reply:
You can add method to `AuthenticationTestHelper` to avoid repitition and it might also be a general useful utility. Alternatively, you can use `AuthenticationTestHelper.builder().internal().build().getUser().principal()`. But it is likely way too verbose.

Reply:
Perfect place for it, thanks for the pointer.